### PR TITLE
proper resize based on screen size

### PR DESCRIPTION
**Pull request by dhananjaylatkar**

### CODE REVIEW

**Comment by Davidy22** [comment]
Still does not account for width, and now crashes when screen is narrow enough

**Comment by dhananjaylatkar** [comment]
> Still does not account for width, and now crashes when screen is narrow enough

It is fixed with recent commit.

File: src/frames.py [hunk 1]
```diff
@@ -64,7 +65,8 @@ def _gallery() -> None:
     def _shoot(self) -> None:
         """Take an image"""
         logger._log_info("Camera was clicked")
-        self.webcam.take_picture_and_save()
+        img_name = f"gallery/Image-{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.jpg"
```
Comment (dhananjaylatkar) [inline src/frames.py:68]: @Trisanu-007 use same format for video files as well.

Comment (Trisanu-007) [reply src/frames.py:68]: Sure its somewhere in the fileIO file .

**Comment by Davidy22** [comment]
It's not centered, but it's smooth enough now

**Pull request merged**